Scripts/Python/nxusBookMachine.py

```diff
@@ -203,9 +203,9 @@
 }
 
 
-kHiddenPersonalAges = ["Personal", "Nexus", "Neighborhood", "city", "AvatarCustomization", "Cleft", "BaronCityOffice", "BahroCave", "PelletBahroCave", "Kveer", "Myst", "LiveBahroCaves", "LiveBahroCave", "Tiam"]
+kHiddenPersonalAges = ["Personal", "Nexus", "Neighborhood", "city", "AvatarCustomization", "Cleft", "BaronCityOffice", "BahroCave", "PelletBahroCave", "Kveer", "Myst", "LiveBahroCaves", "LiveBahroCave", "GoMePubNew", "ChisoPreniv", "VeeTsah", "Serene", "Vothol", "trebivdil"]
 kHiddenCityLinks = ["islmPalaceBalcony03", "KadishGallery", "islmPalaceBalcony02", "islmDakotahRoof", ]
-kHiddenAgesIfInvited = ["BahroCave", "PelletBahroCave", "Pellet Cave", "LiveBahroCave", "LiveBahroCaves", "Myst", "Tiam"]
+kHiddenAgesIfInvited = ["BahroCave", "PelletBahroCave", "Pellet Cave", "LiveBahroCave", "LiveBahroCaves", "Myst", "GoMePubNew"]
```

Comment:
There is no linking panel for the Age "Tiam."

Reply:
Was the Nexus not updated? I know @Hazado was working on that for this release. That version should have the Linking panels, which is why we removed Tiam from the list.

Also note that I just updated these lists to match what's on MOULa.

Reply:
H-uru/moul-assets#104 is blocked due to unknown linking panels being included in the PRPs, which is inappropriate for mainline.

Reply:
> [H-uru/moul-assets#104](https://github.com/H-uru/moul-assets/pull/104) is blocked due to unknown linking panels being included in the PRPs, which is inappropriate for mainline.

I saw the PR [H-uru/moul-assets#104](https://github.com/H-uru/moul-assets/pull/104) was merged. Which panels were included and excluded? What's the latest advice for this portion of the script?